test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -17,3 +20,440 @@ Describe "Windows aliases do not conflict with Linux commands" -Tags "CI" {
         Test-Path Function:more | Should Be $true
     }
 }
+
+
+Describe "Verify approved aliases list" -Tags "CI" {
+    BeforeAll {
+        if ( !$isCoreCLR ) {
```

Comment:
Instead of multiple lists, could we use a hashtable to make it easy to see the differences?  Something like:

```powershell
$aliasList = @{
    "% -> ForEach-Object" = $true
    ...
    "stz -> Set-TimeZone" = $IsWIndows
    "
```

Then iterate through the pairs, only testing the key if the value is `$true`.